server/topic.go

```diff
@@ -964,6 +964,17 @@ func (t *Topic) saveAndBroadcastMessage(msg *ClientComMessage, asUid types.Uid,
 		}
 	}
 
+	if msg.sess != nil && msg.sess.uid != asUid {
+		// Add 'sender' header indicating that message was sent on behalf of another user.
```

Comment:
Can you change the comment to state that:
`sender` is the user who actually sent the message on behalf of `asUid`.

Reply:
Done